data/nmstate/operator.yaml

```diff
@@ -0,0 +1,245 @@
+      labels:
+        app: kubernetes-nmstate
+        component: kubernetes-nmstate-handler
+        name: {{template "handlerPrefix" .}}nmstate-handler
+    spec:
+      # Needed to force vlan filtering config with iproute commands until
+      # future nmstate/NM is in place.
+      # https://github.com/nmstate/nmstate/pull/440
+      hostNetwork: true
+      serviceAccountName: {{template "handlerPrefix" .}}nmstate-handler
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      {{- range $key, $value := .HandlerNodeSelector }}
+      {{"  "}}{{- $key }}: {{  $value }}
+      {{- end}}
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+      containers:
+        - name: nmstate-handler
+          args:
+            - --v=production
+          # Replace this with the built image name
+          image: {{ .HandlerImage }}
+          imagePullPolicy: {{ .HandlerPullPolicy }}
+          command:
+            - kubernetes-nmstate
+          env:
+            - name: WATCH_NAMESPACE
+              value: ""
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: "{{template "handlerPrefix" .}}nmstate"
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: INTERFACES_FILTER
+              valueFrom:
+                configMapKeyRef:
+                  name: {{template "handlerPrefix" .}}nmstate-config
+                  key: interfaces_filter
+            - name: ENABLE_PROFILER
+              value: "False"
+            - name: PROFILER_PORT
+              value: "6060"
+            - name: NMSTATE_INSTANCE_NODE_LOCK_FILE
+              value: "/var/k8s_nmstate/handler_lock"
+          volumeMounts:
+            - name: dbus-socket
+              mountPath: /run/dbus/system_bus_socket
+            - name: nmstate-lock
+              mountPath: /var/k8s_nmstate
+          securityContext:
+            privileged: true
+      volumes:
+        - name: dbus-socket
+          hostPath:
+            path: /run/dbus/system_bus_socket
+            type: Socket
+        - name: nmstate-lock
+          hostPath:
+            path: /var/k8s_nmstate
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{template "handlerPrefix" .}}nmstate-config
+  namespace: {{ .HandlerNamespace }}
+data:
+  interfaces_filter: "veth*"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{template "handlerPrefix" .}}nmstate-webhook
+  namespace: {{ .HandlerNamespace }}
+  labels:
+    app: kubernetes-nmstate
+spec:
+  publishNotReadyAddresses: true
+  ports:
+    - port: 443
+      targetPort: 8443
+  selector:
+    name: {{template "handlerPrefix" .}}nmstate-webhook
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
```

Comment:
You need to list it under test check components https://github.com/kubevirt/cluster-network-addons-operator/blob/de15dd0aa3c79a38a21ccc82d0df900da054abad/test/check/components.go#L21